core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1811,6 +1811,11 @@ public View getView(String name) {
         return viewGroupMixIn.getViews();
     }
 
+    @Exported     //Just to be exported - @Exported on default methods in interface is not supported
```

Comment:
Why export? Perhaps with a lower visibility?

Reply:
I think you're right, @szpak

Reply:
Dummy question, is adding this required for the whole improvement to work? If not, I'd strongly recommend to file it in a separate PR to unblock the PR and have a dedicated discussion. (I suspect it is needed, but then at least this will be clarified, hopefully not just for me)

Reply:
This is not required to make this work, so should be removed. This looks like a separate and mostly unrelated change.

Reply:
removed @daniel-beck